Azure/azure-cli
# [RDBMS] `az mysql flexible-server restore/georestore/replica create`: Add public-access parameter for restore, replication and georestore

**Related command**
az mysql flexible-server restore --public-access
az mysql flexible-server georestore --public-access
az mysql flexible-server replica create --public-access 

**Description**
This PR uses latest Python SDK package which in turn uses the latest GA API version 2022-09-30. This PR will allow customers to enable/disabled public access when creating replica and restore.

**Testing Guide**
Covered by E2E

---

This checklist is used to make sure that common guidelines for a pull request are followed.
